modules/ipi-install-modifying-install-config-for-dual-stack-network.adoc

```diff
@@ -28,9 +28,10 @@ serviceNetwork:
 - fd03::/112
 ----
 
+ifndef::vsphere[]
 [IMPORTANT]
 ====
-If you specified an NMState configuration in the `networkConfig` section of your `install-config.yaml` file, ensure you add `interfaces.wait-ip: ipv4+ipv6` to the NMState YAML file to resolve an issue that prevents your cluster from deploying on a dual-stack network.
+On a bare-metal platform, if you specified an NMState configuration in the `networkConfig` section of your `install-config.yaml` file, ensure you add `interfaces.wait-ip: ipv4+ipv6` to the NMState YAML file to resolve an issue that prevents your cluster from deploying on a dual-stack network.
```

Comment:
```suggestion
On a bare-metal platform, if you specified an NMState configuration in the `networkConfig` section of your `install-config.yaml` file, add `interfaces.wait-ip: ipv4+ipv6` to the NMState YAML file. This  prevents your cluster from deploying on a dual-stack network.
```

Reply:
Just a suggestion, but I believe it removes unnecessary words as well as pointing out an "issue".